### DISCUSSION TITLE
Does anyone has the A3DBinghamLiquid.dll ?

### DISCUSSION THREAD

**Discussion by jacobosaa** [Anura3D/Anura3D_OpenSource#138] [General]
Hello everybody! My name is Jacobo. I'm a civil engineering student currently working on my thesis about using the Material Point Method (MPM) to calibrate rheological parameters in thickened tailings, such as yield stress and viscosity. This involves comparing the movement of columns with different sizes and solid concentrations. I'm also comparing the Mohr-Coulomb Model (which Anura has already implemented) with the Bingham model.

The primary objective is to find a range of solid concentrations where the behavior of the columns, using both models, yields similar results.

When it comes to modeling a column using the Mohr-Coulomb model in Anura, applying fixities, and following the steps outlined in the tutorial manual, I have no issues. However, I've encountered challenges while attempting to use the Bingham model following the same procedure.  I've noticed that the sources file within the program lacks an "A3DBingham.dll" file. During my research, I came across code with that name, but due to my limited coding skills, I'm uncertain whether it would address my problem or not.

At the point where i'm about to export the calculation files (.GOM and .CPS) this message (image) appears, and this is where i'm stuck. I don't know if it is because i dont have that .dll file, or is it because my problemtype folder is not right, or maybe my anura3d version (2019.2) is old. I was looking the tutorial for the 2019.1 version and it has a variation where a column is model as a liquid, but there is no longer that variation in other versions. I'm really hopeless because i haven't found any help for my problem :(.
![Captura de pantalla 2023-09-01 005026](https://github.com/Anura3D/Anura3D_OpenSource/assets/128545000/0904e50a-661a-475b-8eed-6eadbd0766a6)

Hoping someone sees this.

Jacobo

**Comment by luisez1988** [comment]
Hi, 

This might be due to incompatibility between your executable version and the problem type. I recommend getting the latest version and trying to run your models there. You should be able to run the collum collapse problem using the Bingham model.
